docs/source/install-django-project.rst

```diff
@@ -22,161 +22,120 @@ downloading data as a researcher.
    PR to the lookit-api repo to update the documentation! For notes on Linux installation,
    there may be helpful information in a `previous version of invoke tasks.py <https://github.com/lookit/lookit-api/blob/d1b8c9b43cb7d7bda7cdbe5958236d99af42341d/tasks.py>`__.
 
-Django
-~~~~~~
-
-These instructions will get walk you through the installation and running of the Django's local envionment.  
-
-#. Install `Homebrew <https://brew.sh/>`__.  Run update and upgrade:
 
-   .. code-block:: shell
+Prerequisites
+~~~~~~~~~~~~~
 
-      brew update && brew upgrade
+Before you can begin setting your local environment, we'll need to install a couple of things.
```

Comment:
In this section maybe we should add:
- Install homebrew (or does in not matter which package manager someone is using?). Maybe we could suggest this, and then suggest `brew install mkcert` in parentheses for the next step.
- Install git, and make sure that ssh keys are set up (for step 1): https://docs.github.com/en/authentication/connecting-to-github-with-ssh (this was an issue for both me and Melissa: https://github.com/lookit/lookit-docs/issues/230)